unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -469,7 +474,8 @@ public void LoadParcelScenes(string decentralandSceneJSON)
 
             MessagingControllersManager.i.ForceEnqueueToGlobal(MessagingBusId.INIT, queuedMessage);
 
-            TrySortScenesByDistance(forceSort: true);
+            sceneSortDirty = true;
+            TrySortScenesByDistance();
```

Comment:
I feel like we can keep the `true` here, what do you think?

```suggestion
            TrySortScenesByDistance(true);
```